### DISCUSSION TITLE
portal.shadowsocks.nz is invalid..?

### DISCUSSION THREAD

**Discussion by gdkiddd** [shadowsocks/shadowsocks-org#215] [General]
why?  where is the new entrance for shadowsocks user?

**Comment by luoxufeiyan** [comment]
AFAIK, `shadowsocks.nz` or `shadowsocks.com` is not the official service provided by Shadowsocks. You may need to contact the services provider for the service availability issue. 

BTW, this link works for me, which means it might be blocked by GFW.

**Reply by xyyjk** [reply]
https://portal.shadowsocks.au/aff.php?aff=39783

**Comment by LANRRI** [comment]
same as you，have you solved that？

**Reply by gdkiddd** [reply]
I solved it! the new link below:
https://portal.shadowsocks.au/aff.php?aff=52355
